tests/instruments/agilent/test_agilent33500.py

```diff
@@ -291,3 +291,17 @@ def test_ramp_symmetry(ramp_symmetry):
         inst.ch[1].ramp_symmetry = ramp_symmetry
         inst.ch[2].ramp_symmetry = ramp_symmetry
         inst.ramp_symmetry = ramp_symmetry
+
+
+def test_channel_phase_synchronization():
+    """
+    Test Agilent 33500 channel phase synchronization
+    """
+    with expected_protocol(
+        Agilent33500,
+        [
+            ("PHAS:SYNC", '')
+        ]
+    ) as inst:
+        response = inst.phase_sync()
+        assert response is None
```

Comment:
Just as a possibility, how to make it shorter:
```suggestion
        assert inst.phase_sync() is None
```
However, as the method does not return anything, you could drop the assertion altogether and just call `inst.phase_sync()`.

Reply:
Well, the assertion checks that indeed `None` is returned, so would catch it if that should change later. If you just call it, this is not checked. So, I'd keep the assert, your suggestion looks fine to me.